lib/tasks_private/spec_helper.rake

```diff
@@ -48,13 +52,16 @@ class PopulateTower
   # configured_system              (hosts)             : 133
   # inventory_group                (inventories)       : 29
   # credential                     (credentials)       : 47
+  # configuration_script_payload   (playbooks)         : 139
+  #     hello_repo                                     : 61
   #
   require 'faraday'
   require 'faraday_middleware'
 
   MAX_TRIES = 10
   TRY_SLEEP = 2
   DEL_SLEEP = 20
+  TOWER_DATA_YAML = ManageIQ::Providers::AnsibleTower::Engine.root.join('spec', 'support', 'tower_data.yml')
```

Comment:
Remove this line and pass the file path as parameter when dumping to file (Line 11).  This way just from reading the calling code of this class, people would know what to expect.

And you don't have to split the path into 3 parts, you can simply jam them into 1 literal string being appended to the engine root.

Reply:
Sure! About the 3 parts.. I just copy-pasted that without thinking, thanks for pointing that out. :rocket:

Reply:
I don’t like the fact that the path is here in the Rake task. Since the YAML is going to be used by the Specs, I’d prefer to have the constant somewhere along them and use it from here.